### DISCUSSION TITLE
disappearing event ids?

### DISCUSSION THREAD

**Discussion by rlrl** [blakeblackshear/frigate#1859] [Ask A Question] [answered]
i'm capturing events as they come across mqtt for particular zones. but when i go back & try to view the thumbnail or the video for some events, they don't exist. do they merged into other events? i notice there are multiple ids for the same object sometimes. thanks

**Comment by blakeblackshear** [comment, marked as answer]
Yes. There is some logic to go back and merge some events that overlap every 5 minutes or so.

**Reply by doghouselabs** [reply]
thanks. is there a way to know which id was merged into which?

**Reply by blakeblackshear** [reply]
Not that I can think of

**Reply by tve** [reply]
ouch, might be nice to make this configurable. Or leave some trace in the DB so a request for the merged event can return some form of redirect...
